### PR TITLE
Add EIP-5560: Redeemable NFTs

### PR DESCRIPTION
**Pull request by julienbrg**

### CODE REVIEW

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Please fix the CI errors: https://github.com/ethereum/EIPs/pull/5560/files

File: EIPS/eip-5560.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+eip: 5560
+title: Redeemable NFTs
+description: Links physical objects to NFTs
```
Comment (xinbenlv) [inline EIPS/eip-5560.md:4]: [Editorial] It is hard for me to connect the description with the title, could you help elaborate the "redeemable" aspect in the description?

File: EIPS/eip-5560.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Motivation
+
+An increasing amount of NFT issuers such as artists, fine art galeries, auction houses, brands and others want to offer a physical object to the holder of a given NFT. This standard allows EIP-721 NFTs to signal reedemability.
+
+## Specification
+
+_The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119._
+
+**EIP-721 compliant contracts MAY implement this EIP to provide a standard method of receiving information on redeemability.**
```
[the remark this answers is not in the document]
Comment (julienbrg) [reply EIPS/eip-5560.md:26]: It's not necessary.

File: EIPS/eip-5560.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+```solidity
+import '@openzeppelin/contracts/utils/introspection/ERC165.sol';
+
+/**
+ * @dev Implementation of Redeemable for ERC-721s
+ *
+ */
+
+interface IRedeemable is ERC165 {
```
Comment (xinbenlv) [inline EIPS/eip-5560.md:44]: [Technical, peer-review, optional] Consider add a "event" for "Redeemed" when a redemption occurs?

File: EIPS/eip-5560.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+```solidity
+require(ownerOf(tokenId) == msg.sender, "ERC721Redeemable: You are not the owner of this token");
+```
+
+After the `redeem()` function is triggered, `isRedeemable()` function returns `false`.
+
+### `Redeem` event
+
+When the `redeem()` function is triggered, the following event is emitted:
```
Comment (xinbenlv) [inline EIPS/eip-5560.md:79]: [Editorial] 1. also include this event def into the required interface above.
2. clarify if it's MUST or SHOULD/MAY
```suggestion
When the `redeem()` function is triggered, the following event **MUST** be emitted:
```

File: EIPS/eip-5560.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (julienbrg) [reply EIPS/eip-5560.md]: Done.

File: EIPS/eip-5560.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-30
+requires: 165, 721
+---
+
+## Abstract
+
+The Redeemable NFT Extension adds a `redeem` function to [EIP-721](./eip-721.md). It can be implemented when an NFT issuer wants his/her NFT to be redeemed for a physical object.
```
[the remark this answers is not in the document]
Comment (julienbrg) [reply EIPS/eip-5560.md]: Thanks for you help!

File: EIPS/eip-5560.md (hunk 1 above)
Comment (julienbrg) [reply EIPS/eip-5560.md]: Thanks, I suggested "Makes an NFT redeemable for a physical object".

File: EIPS/eip-5560.md (hunk 3 above)
Comment (julienbrg) [reply EIPS/eip-5560.md:44]: Good point. Thanks again, we added it in the interface.

File: EIPS/eip-5560.md (hunk 4 above)
Comment (julienbrg) [reply EIPS/eip-5560.md]: Well spotted. It's 'MUST'.

**Review by xinbenlv — APPROVED** [review]
LGTM.

**Review by Pandapip1 — APPROVED** [review]
LGTM too

**Pull request merged**